src/main/java/org/junit/runners/model/FrameworkMethod.java

```diff
@@ -149,6 +149,11 @@ public boolean isShadowedBy(FrameworkMethod other) {
     }
 
     @Override
+    boolean isBridgeMethod() {
```

Comment:
@kcooney -- i found an use-case where "isBridge()" is not enough. I think, it should be widened to "isSynthetic()". See https://github.com/junit-team/junit5/compare/ignore_synthetic_methods with a test case included.